repository/src/test/java/org/alfresco/repo/search/impl/solr/DbOrIndexSwitchingQueryLanguageTest.java

```diff
@@ -159,7 +161,7 @@ public void hybridSearchWhenNoIndexLanguageAvailable()
         queryLang.executeQuery(searchParameters);
     }
 
-    @Test(expected=DisabledFeatureException.class)
+    @Test(expected= DisabledFeatureException.class)

```

Comment:
Is the added space just a typo?

Reply:
Hmm - may have been the IDE when refactoring (during rename). 

I'll fix it thanks (ie. one less diff) 👍

Reply:
Hmm - looks like a spurious hidden whitespace character (disappears with ?w=1).

I'll double-check

Reply:
Fixed, thanks